# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fbterm/hotfix-select.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fbterm-1.8/src/lib/io.h.vanilla	2025-07-09 23:20:01.962488050 +0200
+++ fbterm-1.8/src/lib/io.h	2025-07-09 23:20:07.968963052 +0200
@@ -23,6 +23,7 @@
 
 #include "type.h"
 #include "instance.h"
+#include <sys/select.h>
 
 class IoPipe {
 public:
